src/test/java/org/shikshalokam/backend/ep/TestUserRoleExtensionCreate.java

```diff
@@ -0,0 +1,92 @@
+package org.shikshalokam.backend.ep;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.shikshalokam.backend.PropertyLoader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.net.URI;
+
+/*
+Validating the User Role Extension Scenarios in this below class
+ */
+public class TestUserRoleExtensionCreate extends ElevateProjectBaseTest {
+    static final Logger logger = LogManager.getLogger(TestUserRoleExtensionCreate.class);
+    public static String USER_ROLE_CREATE_ENDPOINT = PropertyLoader.PROP_LIST.getProperty("elevate.createuserroleext.endpoint");
+    public static String BASE_URL = PropertyLoader.PROP_LIST.getProperty("elevate.qa.api.base.url");
+    public static String INTERNAL_ACCESS_TOKEN = "Fqn0m0HQ0gXydRtBCg5l";
+
+    @Test(description = "Validating user role extension creation with valid payload")
+    public void testUserRoleCreateWithValidPayload() {
+        ElevateProjectBaseTest.loginToElevate(PropertyLoader.PROP_LIST.getProperty("elevate.login.contentcreator"), PropertyLoader.PROP_LIST.getProperty("elevate.login.contentcreator.password"));
+        String validPayload = "{\"userRoleId\": \"88688\", " +
+                "\"title\": \"QAA\", " +
+                "\"entityTypes\": [{\"entityType\": \"state\", " +
+                "\"entityTypeId\": \"6672ce0fc05aa58f89ba12f1\"}]}";
+
+        // Taking the response from the user role creation API using method calling
+        ElevateProjectBaseTest.response = createUserRoleExtension(validPayload);
```

Comment:
why are we assigning to  ElevateProjectBaseTest.response ?
Need to use local variable and proceed , because  response is specific to your test .
Same comment for all the Test Mentods